### ISSUE TITLE
Disc 11 playing alongside custom disc

### ISSUE THREAD

**Issue by MAX2136** [TeamTernate/infinite-music-discs#162]
I added a custom disc, but a little bit after it starts playing, disc 11 starts to play as well. Minecraft version 1.20.1, only this resourcepack and datapack are active. Sorry if I'm just dumb and doing something wrong.

**Comment by GuentherBasart** [comment]
Yeah I have the same issue for some reason. I'm also playing on 1.20.1 but with a few mods installed (150)

**Comment by zmarotrix** [comment]
Same issue. On 1.20.1.

**Comment by nickdibs01** [comment]
I have the same issue on 1.20.1. All music stops when I remove the disk, though.
